lib/KettleMultiConfig.js

```diff
@@ -0,0 +1,94 @@
+/*
+Kettle Multi-Configuration support
+
+Copyright 2015 OCAD University
+Copyright 2015 Raising the Floor - International
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/kettle/blob/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    kettle = fluid.registerNamespace("kettle");
+
+fluid.defaults("kettle.multiConfig.config", {
+    gradeNames: ["fluid.component", "{that}.buildMultiKettleConfig"],
+    invokers: {
+        buildMultiKettleConfig: {
+            funcName: "kettle.multiConfig.build",
+            args: "{that}.options.configs"
+        }
+    }
+});
+
+kettle.multiConfig.template = {
+    components: {
+        server: {
+            type: "fluid.component",
+            options: {
+                components: null, // To be filled in
+                events: {
+                    onListen: {
+                        events: null // To be filled in
+                    },
+                    onStopped: {
+                        events: null // To be filled in
+                    }
+                },
+                invokers: {
+                    stop: {
+                        funcName: "kettle.multiConfig.stopServers",
+                        args: null // To be filled in
+                    }
+                }
+            }
+        }
+    }
+};
+
+kettle.multiConfig.build = function (servers) {
```

Comment:
All failures for this file were due to its test driver not being included in all-tests.js